capstone/files/enrichment/virustotal.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+# Copyright (C) 2015-2020, Wazuh Inc.
+# October 19, 2017.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+# Wazuh, Inc <support@wazuh.com>
+
+import json
+import sys
+import time
+import os
+from socket import socket, AF_UNIX, SOCK_DGRAM
+
+try:
+    import requests
+    from requests.auth import HTTPBasicAuth
+except Exception as e:
+    print("No module 'requests' found. Install: pip install requests")
+    sys.exit(1)
+
+try:
+    import vt
+except Exception as e:
+    print("No module 'vt' found. Install pip install vt-py")
```

Comment:
You can check the module's name in the exception object:

```python
>>> try:
...     import blablabla
... except ImportError as e:
...     print(e.name)
... 
blablabla
```

So you can refactor all imports in a single try-except block